RecoTauTag/RecoTau/src/AntiElectronIDMVA6.cc

```diff
@@ -1269,3 +1324,29 @@ double AntiElectronIDMVA6::dCrackEta(double eta)
 
   return std::abs(retVal);
 }
+
+bool AntiElectronIDMVA6::atECalEntrance(const reco::Candidate* part, math::XYZPoint &pos)
+{
+  bool result = false;
+  double bField = 3.8; // hopefully Teslas are proper units (better to take it from event setup) 
```

Comment:
I see that the config has no GlobalTag settings. Something relevant/standard for the input sample should be used.